tokens/ETH.json

```diff
@@ -631,6 +631,22 @@
     "decimals": 18,
     "logoURI": "https://raw.githubusercontent.com/parallel-protocol/parallel-brand-kit/main/Tokens/USDp/sUSDp.svg"
   },
+  {
+    "name": "Staked PRL 1",
+    "address": "0xeAd729472f82E5eC2FF4e691d67633077C1B5901",
+    "symbol": "sPRL1",
+    "chainId": 1,
+    "decimals": 18,
+    "logoURI": "https://raw.githubusercontent.com/parallel-protocol/parallel-brand-kit/main/Tokens/PRL/sPRL.svg"
+  },
+  {
+    "name": "Staked PRL 2",
```

Comment:
Please use the correct name: https://etherscan.io/token/0xE8A2d848fE656E34A6caA35f375B42979e322135